datanode/entities/proposal.go

```diff
@@ -56,6 +57,8 @@ func (p *ProposalType) String() string {
 		return "updateNetworkParameter"
 	case ProposalTypeNewFreeform:
 		return "newFreeform"
+	case ProposalTypeSuccessorMarket:
```

Comment:
Not sure about this one, a successor market is essentially a market proposal, just one with 2 fields not being nil. The dedicated proposal type was removed, so I don't think the datanode should differentiate between a market proposal with/without a parent market ID

Reply:
Yes, sorry, this should have been removed after the removal of a separate successor market proposal but I missed it. Will fix.